GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -146,13 +146,15 @@ class ChatView: EngagementView {
         tableView.estimatedRowHeight = UITableView.automaticDimension
         tableView.separatorStyle = .none
         tableView.contentInset = kChatTableViewInsets
+        tableView.keyboardDismissMode = .onDrag
```

Comment:
Hm, we already have such functionality achieved through implementing `scrollViewWillBeginDragging(_ scrollView: UIScrollView)` from `UITableViewDelegate` (lines 379-381).

I tried removing that delegate method and leaving just this new line: it works, but the animation looks a bit strange (suggestions strip stays for some time). Just like in [this SO question](https://stackoverflow.com/questions/66180854/why-the-animation-of-keyboarddismissmode-is-weird). Funnily enough, the answer proposes to do what we are already doing 🤷

Reply:
Good catch, will remove :)